src/locales/en-US/admin.ftl

```diff
@@ -1541,10 +1574,103 @@ dashboard-heading-last-updated = Last updated:
 dashboard-today-heading = Today's activity
 dashboard-today-new-comments = New comments
 dashboard-alltime-new-comments = All time total
+dashboard-alltime-new-comments-archiveEnabled = { $value } { $unit ->
+    [second] { $value ->
+      [1] second
+      *[other] seconds
+    }
+    [minute] { $value ->
+      [1] minute
+      *[other] minutes
+    }
+    [hour] { $value ->
+      [1] hour
+      *[other] hours
+    }
+    [day] { $value ->
+      [1] day
+      *[other] days
+    }
+    [week] { $value ->
+      [1] week
+      *[other] weeks
+    }
+    [month] { $value ->
+      [1] month
+      *[other] months
+    }
+    [year] { $value ->
+      [1] year
+      *[other] years
+    }
+    *[other] unknown unit
+  } total
 dashboard-today-rejections = Rejection rate
 dashboard-alltime-rejections = All time average
+dashboard-alltime-rejections-archiveEnabled = { $value } { $unit ->
+    [second] { $value ->
+      [1] second
+      *[other] seconds
+    }
+    [minute] { $value ->
+      [1] minute
+      *[other] minutes
+    }
+    [hour] { $value ->
+      [1] hour
+      *[other] hours
+    }
+    [day] { $value ->
+      [1] day
+      *[other] days
+    }
+    [week] { $value ->
+      [1] week
+      *[other] weeks
+    }
+    [month] { $value ->
+      [1] month
+      *[other] months
+    }
+    [year] { $value ->
+      [1] year
+      *[other] years
+    }
+    *[other] unknown unit
+  } average
 dashboard-today-staff-comments = Staff comments
 dashboard-alltime-staff-comments = All time total
+dashboard-alltime-staff-comments-archiveEnabled = { $value } { $unit ->
```

Comment:
I really want to do this! I've tried to nest Localizations in the past, FTL doesn't seem to allow it. It's kind of annoying and I wish I could do it, but I've never successfully managed to do so.

Reply:
Ah...hmm. Would it work to do something like two `span`s beside each other, since that wouldn't be nesting the localizations?

Reply:
I try not to do that for when someone localizes based on a language that goes right to left instead of left to right. I want them to be able to localize it seamlessly as one `<Localized>` object that they just update the text + values for.